app/e2e/utils.ts

```diff
@@ -0,0 +1,54 @@
+import { Locator, Page, expect } from "@playwright/test";
+
+export const getContentLocator = async (page: Page) => {
+  return page.getByTestId("content");
+};
+
+export const getBreadcrumbLocator = async (page: Page) => {
+  return page.getByTestId("breadcrumb");
+};
+
+export const packetGroupNameFromListItem = async (listItem: Locator) => {
+  return await listItem.getByRole("heading").innerText();
+};
+
+export const getFirstPacketGroupListItem = async (content: Locator) => {
+  return content.getByRole("listitem").first();
+};
+
+export const getDisplayString = (id: string) => id.replaceAll("-", " ");
+
+export const navigateToFirstPacketGroup = async (content: Locator) => {
+  const firstPacketGroup = await getFirstPacketGroupListItem(content);
+  const firstPacketGroupName = await packetGroupNameFromListItem(firstPacketGroup);
+  await firstPacketGroup.getByRole("heading").click();
+  return firstPacketGroupName;
+};
+
+export const navigateToFirstPacketGroupLatestPacket = async (content: Locator) => {
+  const firstPacketGroup = await getFirstPacketGroupListItem(content);
+  const packetGroupName = await packetGroupNameFromListItem(firstPacketGroup);
+  const latestLink = firstPacketGroup.getByRole("link", { name: "Latest" });
+  const packetId = (await latestLink.getAttribute("href")).split("/").at(-1) as string;
+  await latestLink.click();
+  return { packetGroupName, packetId };
+};
+
+export const getPacketPageAccordionSection = async (content: Locator, title: string, clickToExpand = false) => {
+  const section = content.locator("div.border-b:has(h3[data-orientation='vertical'])").filter({ hasText: title });
+  await expect(await section).toBeVisible();
+  if (clickToExpand) {
+    await section.locator("h3[data-orientation='vertical']").click();
+  }
+  return section;
+};
+
+export const selectPacketPageTab = async (content: Locator, tab: string) => {
+  const nav = await content.getByRole("navigation");
+  await nav.getByRole("link", { name: tab }).click();
+};
+
+export const getPackitInstance = (baseURL: string) => {
```

Comment:
At first I was a bit confused why this would be needed, and then I saw the code that uses it to find the instance-relative path. I would suggest changing this function to doing that operation here instead, especially since it will come up in more places.

```js
export const getRelativePath = (baseURL: string, href: string) {
    const basePath = new URL(baseURL).pathname;
    return href.startsWith(basePath) ? basePath.slice(path.LENGTH) : href;
}
```

Reply:
It will do. Am changing to something similar to this.